lib/complate.js

```diff
@@ -0,0 +1,44 @@
+let { VirtualBundle } = require("faucet-pipeline-js/lib/bundle/virtual");
```

Comment:
This entire file would be a plugin IMHO.

Reply:
Whaaaait a second. I mean the file "lib/complate.js" should not be part of the styleguide NPM package, but be a separate "styleguide-complate" package. Keep it in faucet-pipeline-js, it's fine 👍

Reply:
Oh, you really set me up there by commenting this particular line, making me go on that diatribe - well played sir...

So yeah, I agree, but it doesn't seem particularly urgent?

Reply:
Not urgent, but I want to keep that in mind when thinking about which parts are in that file vs. other parts (for example registering the template engine in the highlighter).